zcash_keys/src/keys.rs

```diff
@@ -71,6 +70,33 @@ pub mod sapling {
     }
 }
 
+/// A hash of a seed used for an HD account.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub struct HDSeedFingerprint([u8; BLAKE2B_LENGTH]);
```

Comment:
Per [the Rust naming conventions](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html) (along with the bugfix):
```suggestion
pub struct HdSeedFingerprint([u8; 32]);
```